root/etc/supervisor.conf

```diff
@@ -15,14 +15,14 @@ daemon=off
 priority = 2
 
 [program:webapp]
-environment=NODE_ENV="production",PORT=3000
+environment=NODE_ENV="production",PORT="%(ENV_WEB_APP_PORT)s",HTTPS_PROXY="%(ENV_HTTPS_PROXY)s",HTTP_PROXY="%(ENV_HTTP_PROXY)s",NO_PROXY="%(ENV_NO_PROXY)s"
 command=/usr/bin/node app.js
 user=nbxyz
 directory=/app
 priority = 3
 
-[program:in.tftpd]
-command=/usr/sbin/in.tftpd -Lvvv --user nbxyz --secure %(ENV_TFTPD_OPTS)s /config/menus
+[program:dnsmasq]
+command=/usr/sbin/dnsmasq --port=0 --keep-in-foreground --enable-tftp --user=nbxyz --tftp-secure --tftp-root=/config/menus %(ENV_TFTPD_OPTS)s
```

Comment:
For simpler approach - this is a follow-up of changes made in docker netbootxyz repository